CHANGES.md

```diff
@@ -4,13 +4,23 @@ This file is only intended to serve the developers of this repository and of the
 
 This provides an overview of the releases that have been made using this release pipeline.
 
-## Work In Progress - March: 2023: Security release 1.39
+## Work In Progress - April: 2023: 1.39.3 Security release
 
-For a MediaWiki update to 1.39.2
+- [MediaWiki security release announcement](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/message/6UQBHI5FWLATD7QO7DI4YS54U7XSSLAN/)
+- [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.39)
 
 | Suite Version | MediaWiki release | Date available | Run number | Release task  |
 |---------------|-------------------|----------------|------------| --------------|
-| wmde.12 ([spec](),[metadata]()) | 1.39.2    | March 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
+| wmde.13 ([spec](),[metadata]()) | MediaWiki (1.39.3, PHP 8.0, PHP 8.1, PHP 8.2) | April 2023 | ??? | [T332786](https://phabricator.wikimedia.org/T332786) |
```

Comment:
As in https://github.com/wmde/wikibase-release-pipeline/pull/415 the PHP version here should bethe version actually in the container.

Reply:
as per https://hub.docker.com/layers/library/php/8.1-apache/images/sha256-5ffb8acb1697f09cbad85693fbf6c16ed107c3123cfdd841652cb7f400f8c66f?context=explore we could add debian/apache2 2.4.57-2

Reply:
as per https://github.com/wikimedia/mediawiki-docker/blob/5dfcc29a3820d6129124f8eb154a6632d61a3162/1.39/apache/Dockerfile and https://hub.docker.com/layers/library/php/8.1-apache/images/sha256-5ffb8acb1697f09cbad85693fbf6c16ed107c3123cfdd841652cb7f400f8c66f?context=explore it should be PHP 8.1.20